src/app/screens/Publishers/websites.json

```
@@ -96,27 +96,39 @@
     "items": [
       {
         "title": "WebLN Experiments",
-        "subtitle": "A collection of Lightning web apps",
+        "subtitle": "A collection of Lightning web widgets",
         "logo": "https://cdn.getalby-assets.com/alby-extension-website-screen/twentyuno.svg",
         "url": "https://webln.twentyuno.net/"
       },
       {
+        "title": "Bolt.Fun",
+        "subtitle": "A fun directory of Lighting web apps",
+        "logo": "https://cdn.getalby-assets.com/alby-extension-website-screen/bolt-fun_thumbnail.svg",
+        "url": "https://makers.bolt.fun/"
+      },
+      {
+        "title": "Geyser",
+        "subtitle": "Crowdfunding projects with Bitcoin Lightning",
+        "logo": "https://cdn.getalby-assets.com/alby-extension-website-screen/geyser_thumbnail.svg",
+        "url": "https://geyser.fund/"
+      },
+      {
+        "title": "Amboss Space",
```

```suggestion
        "title": "Amboss",
```
isn't that only `Amboss`?